sound/core/init.c

```diff
@@ -695,9 +695,21 @@ card_number_show_attr(struct device *dev,
 
 static DEVICE_ATTR(number, 0444, card_number_show_attr, NULL);
 
+static ssize_t
+components_show(struct device *dev,
```

Comment:
please merge the above two lines.

Reply:
keeping same layout as other functions in this file.

Reply:
@plbossart in sound/core/init.c I see exactly 3 functions, following the style, that you're using here: `card_id_show_attr()`, `card_id_store_attr()`, and `card_number_show_attr()`. All the other many functions in that file have the function name on the same line as the function type. But those are the 3 functions dealing with device attributes and you're adding one more, so, seems like it fits.